CryptoWallet/Data/Model/AssetInfo.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  AssetInfo.swift
+//  CryptoWallet
+//
+//  Created by Phan Dinh Van on 4/5/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct AssetInfo {
+    var assetLogo: UIImage?
+    var assetName = ""
+    var assetSymbol = ""
+    var assetPrice = 0.0
+    var asset24hChange = 0.0
+    var assetAmount = 0.0
```

Comment:
bỏ hết chữ asset đi đặt tên được r
```suggestion
    var amount = 0.0
```